nemo/collections/asr/data/audio_to_text.py

```diff
@@ -288,7 +288,10 @@ def cache_data(manifest_filepaths, cache_audio, num_workers, max_num_workers):
                         with open(cached_manifest_file, 'r') as f:
                             for line in f:
                                 item = json.loads(line)
-                                store_path = os.path.join(manifest_dir, item['audio_filepath'])
+                                if not is_datastore_path(item['audio_filepath']):
```

Comment:
@pzelasko Was this supposed to be the PR that allows use of tar datasets without full coverage in the manifest?